api/src/app.controller.ts

```diff
@@ -27,10 +28,14 @@ export class AppController {
 
   @Get('config')
   @Public()
+  @AddHiddenFlag(() => {
```

Comment:
Programmatically nice but I think this is an added overhead for simple process.  We normally create this kind of component if there are more than one use of such functionality.